doc/HACKING.md

```diff
@@ -98,6 +98,29 @@ will print out (to stderr) a command such as:
 Run this command to start debugging. You may need to type `return` one more time
 to exit the infinite while loop, otherwise you can type `continue` to begin.
 
+Database
+--------
+
+c-lightning state is persisted in `lightning-dir`. It is a sqlite database
+stored in the `lightningd.sqlite3` file, typically under `~/.lightning`. You can
+run queries against this file like so:
+
+    $ sqlite3 ~/.lightning/lightningd.sqlite3 "SELECT hex(prev_out_tx), prev_out_index, status FROM outputs"
+
+or you can launch into the sqlite3 repl and check things out from there:
+
+    $ sqlite3 ~/.lightning/lightningd.sqlite3
+    SQLite version 3.21.0 2017-10-24 18:55:49
+    Enter ".help" for usage hints.
+    sqlite> .tables
+    channel_configs  invoices         peers            vars
+    channel_htlcs    outputs          shachain_known   version
+    channels         payments         shachains
+    sqlite> .schema outputs
+    ...
+
+Some data is stored as raw bytes, use `hex(column)` to pretty print these.
```

Comment:
Nit: Uppercase `HEX(column)` to indicate SQL function.

Reply:
:+1: